docs/ext/bridge/api.rst

```diff
@@ -157,3 +157,9 @@ BridgeContext Subclasses
 .. data:: discord.ext.bridge.Context
 
     Alias of :data:`typing.Union` [ :class:`.BridgeExtContext`, :class:`.BridgeApplicationContext` ] for typing convenience.
+
+Option
+------
+
+BridgeOption
+~~~~~~~~~~~~
```

Comment:
We're able to preview docs [here](https://pycord--2252.org.readthedocs.build/en/2252/ext/bridge/api.html#option).
```suggestion
~~~~~~~~~~~~

.. attributetable:: discord.ext.bridge.BridgeOption

.. autoclass:: discord.ext.bridge.BridgeOption
    :members:
```